_data/sw_servers.yml

```diff
@@ -105,9 +105,9 @@
           starttls:
           userhost-in-names:
           webirc:
-    - name: Oragono
-      # ref: https://oragono.io/specs.html
-      link: https://oragono.io/
+    - name: Ergo
```

Comment:
If you could move this so that it's just after Bitlbee that'd sort out the alphabetical order